netbout-client/src/test/java/com/netbout/client/NbRule.java

```diff
@@ -46,7 +46,7 @@
      * Get user.
      * @return User
      */
-    public User get() {
+    public static User get() {
```

Comment:
@mkordas Either we have static method, either NonStaticMethodCheck violation. And it is used for tests..